dev/tests/integration/testsuite/Magento/Checkout/ViewModel/CartTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Checkout\ViewModel;
+
+use Magento\Framework\App\Config\MutableScopeConfigInterface;
+use Magento\Framework\ObjectManagerInterface;
+use Magento\Store\Model\ScopeInterface;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test for clear shopping cart config
+ *
+ * @package Magento\Checkout\ViewModel
```

Comment:
Tag @package MUST NOT be used. That is why static test build failed

Reply:
Removed